### ISSUE TITLE
Set dark background color

### ISSUE THREAD

**Issue by jlaw9** [Murali-group/GraphSpace#291]
This is currently a problem because nodes with a white background are not visible (see this [example](http://graphspace.org/graphs/22413))

Looks like the cytoscape style JSON only sets the styles of nodes and edges. The [tokyo-railways example](https://github.com/cytoscape/cytoscape.js/tree/master/documentation/demos/tokyo-railways) sets the background color on line 15 of [style.css](https://github.com/cytoscape/cytoscape.js/blob/master/documentation/demos/tokyo-railways/style.css#L15). 

Can we add the ability to change the background color either in the UI, through the Rest API, or both? I think the simplest option would be to add a "canvas-background-color" option recognized by GraphSpace to the style JSON file being uploaded/downloaded.

**Comment by mitchwagner** [comment]
I agree, I think it would be better from a user perspective to have the option to set it permanently. But, do we want to modify/add non-standard options to the cytoscape style JSON format we use?

Are we already doing that, or would this make us incompatible? Would adding such an option complicate export from GraphSpace?

**Comment by jlaw9** [comment]
It shouldn't be an issue to add additional properties to the top-level of the cytoscape-style JSON. GraphSpace already adds a couple. Other applications that don't use that field/option would just ignore it.

**Comment by jlaw9** [comment]
The css field in GraphSpace which specifies the background color is in [#cyGraphContainer](https://github.com/Murali-group/GraphSpace/blob/e30eed4fbc053e4f79918c040db15c91282e9f9e/static/css/graphspace.css#L138)

**Comment by JingVT** [comment]
[The CytoscapeJS and Cytoscape projects have diverged quite a bit over the years caused this issue](https://code.cytoscape.org/redmine/issues/4317#change-15944). We will wait for them to align the visual styles. The current better solution is suggesting GraphSpace users don't set background color of nodes to white.

**Comment by tmmurali** [comment]
@jlaw9 Can we close this issue now as per @JingVT's comment?

**Comment by jlaw9** [comment]
The cytoscape team said they are not planning to add background-color to the style JSON, so I don't think we should wait for them. 

I still think we should be able to update GraphSpace to recognize when a special field is present at the top level of the style JSON such as "background-color" and then take the value from there and set the corresponding CSS <div> tag to that color value. 

Once we have that working in GraphSpace, we can update CyGraphSpace to grab the background color from Cytoscape and set the "background-color" field in the style JSON before uploading to GraphSpace. That way any graph uploaded from CyGraphSpace will retain the background color in GraphSpace.

We could also/alternatively add an option to the layout/style editor to be able to set the background color in the UI.

**Comment by tmmurali** [comment]
The main concern about your approach is that it introduces yet another GraphSpace-specific style element to the JSON. I would very much like to avoid doing this.
